scripts/sous/theme-build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd web/themes/custom/sous-project
```

Comment:
I could see this file being useful for CI environments... Maybe we should consider doing some additional logic here of "if lando command exists" if not then run `npm` stuff directly.

Reply:
Eventually we can look into this but for the moment we officially support lando.  And rather than looking into "not supporting lando" we should probably look into what else we do support and take a look at it's needs specifically.